### ISSUE TITLE
support specifying path while opening joshuto

### ISSUE THREAD

**Issue by notjedi** [kamiyaa/joshuto#182]
it would be nice to have something like `joshuto <PATH>`. currently this does not work and results in the following error:
```console
error: Found argument `<PATH>` which wasn't expected, or isn't valid in this context

USAGE:
    joshuto [OPTIONS]

For more information try --help
```

**Comment by Vinegret43** [comment]
@notjedi, you actually can specify startup path, you just have to use `--path` option. Though, in my opinion, it's a strange design decision, and i agree that it would be great to make this argument positional instead. This may potentially break some people's shell scripts which rely on this option, but since Joshuto is still in heavy development, this shouldn't be too much of a concern. Would be nice to hear @kamiyaa's opinion on this

**Comment by notjedi** [comment]
@Vinegret43 oh didn't read the man page, my bad. but yeah i would have still opened this issue even if i had read the man page as it makes more sense to specify that path without the `--path` option.

**Comment by kamiyaa** [comment]
Closed via [eb8b155](https://github.com/kamiyaa/joshuto/commit/eb8b155329b2c2689cc5b854f0b87f4448485c66)
